engine.go

```diff
@@ -168,11 +168,19 @@ func (e *engine) render() {
 func (e *engine) write() {
 	if *e.env.getArgs().Eval {
 		fmt.Printf("PS1=\"%s\"", e.renderer.string())
-		if e.rprompt != "" && e.env.getShellName() == zsh {
+		if e.env.getShellName() == zsh {
 			fmt.Printf("\nRPROMPT=\"%s\"", e.rprompt)
 		}
 		return
 	}
+
+	if e.rprompt != "" && (e.env.getShellName() == pwsh || e.env.getShellName() == powershell5) {
```

Comment:
It's perfectly fine like this for now, I'm going to refactor this function a bit afterwards.